scripts/deploy-rpc-playbooks.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+set -e -u -x
+set -o pipefail
+
+export DEPLOY_HAPROXY=${DEPLOY_HAPROXY:-"no"}
+export DEPLOY_ELK=${DEPLOY_ELK:-"yes"}
+export DEPLOY_MAAS=${DEPLOY_MAAS:-"no"}
+export RPCD_DIR=${RPCD_DIR:-"/opt/rpc-openstack/rpcd"}
+export BASE_DIR=${BASE_DIR:-"/opt/rpc-openstack"}
+
+source ${BASE_DIR}/scripts/functions.sh
+
+# begin the RPC installation
+cd ${RPCD_DIR}/playbooks/
+
+# configure everything for RPC support access
+run_ansible rpc-support.yml
+
+# configure the horizon extensions
+run_ansible horizon_extensions.yml
+
+# deploy and configure RAX MaaS
+if [[ "${DEPLOY_MAAS}" == "yes" ]]; then
+  run_ansible setup-maas.yml
+  run_ansible verify-maas.yml
+fi
+
+# deploy and configure the ELK stack
+if [[ "${DEPLOY_ELK}" == "yes" ]]; then
+  run_ansible setup-logging.yml
+
+  # deploy the LB required for the ELK stack
+  if [[ "${DEPLOY_HAPROXY}" == "yes" ]]; then
+    run_ansible haproxy.yml
+  fi
+fi
```

Comment:
Should this script be able to be run independently?  I see benefit in that, but then it means having another place where default variables need to be maintained (currently 3 places).